x-pack/plugins/rule_registry/server/event_log/elasticsearch/index_writer.ts

```diff
@@ -60,11 +63,16 @@ export class IndexWriter {
     }
   }
 
+  public async indexManyNow(docs: Document[]): Promise<estypes.BulkResponse | undefined> {
```

Comment:
What about renaming the methods of this class to make things a little bit more clear?
`indexManyNow` -> `indexMany`
`indexOne` -> `enqueueOne`
`indexMany` -> `qnqueueMany`